tests/epyccel/test_epyccel_sets.py

```diff
@@ -485,6 +525,57 @@ def union_int():
     assert python_result[0] == pyccel_result[0]
     assert set(python_result[1:]) == set(pyccel_result[1:])
 
+def test_temporary_set_intersection(language):
```

Comment:
Are you sure you meant to test all 3 languages?

Reply:
From memory the argument should be:
- `language` for Python/C/Fortran
- `stc_language` for Python/C
- `python_only_language` for Python

Reply:
> Are you sure you meant to test all 3 languages?

Just for python for now.

Reply:
That might explain why you have failing tests then. This is testing all 3 languages. You can see which tests are failing if you run pytest with `-v`